gradle.properties

```diff
@@ -1,7 +1,6 @@
 org.gradle.warning.mode=none
 org.gradle.parallel=true
-org.gradle.jvmargs=-Xmx3g -XX:+HeapDumpOnOutOfMemoryError -Xss2m
-options.forkOptions.memoryMaximumSize=2g
```

Comment:
This was removed since it doesn't actually do anything (it's not a standard Gradle option) and there are no references to a project property of this name.